kolibri/core/discovery/api.py

```diff
@@ -1,21 +1,32 @@
 from rest_framework import viewsets
 from rest_framework.response import Response
 
+from .models import DynamicNetworkLocation
 from .models import NetworkLocation
+from .models import StaticNetworkLocation
 from .serializers import NetworkLocationSerializer
 from kolibri.core.content.permissions import CanManageContent
-from kolibri.core.device.permissions import UserHasAnyDevicePermissions
-from kolibri.core.discovery.utils.network.search import get_available_instances
+from kolibri.core.discovery.utils.network.search import run_peer_discovery
 
 
 class NetworkLocationViewSet(viewsets.ModelViewSet):
     permission_classes = (CanManageContent,)
     serializer_class = NetworkLocationSerializer
-    queryset = NetworkLocation.objects.all()
+    queryset = NetworkLocation.objects
```

Comment:
Does this work? `NetworkLocation.objects` is a Manager instance, not a queryset.

Reply:
It does work, not sure why it didn't seem to work the other way... I'll have to double check on it after deleting pyc files